MAINTAINERSHIP.md

```diff
@@ -0,0 +1,70 @@
+# Request For Maintainership
+
+This document details the acceptance process for requests from external contributors who wish for the `ibc-go` core team to take over **maintainership** of a complex block of code (ie: an application module or a light client implementation). It is the process we also follow internally for features that we develop which will go into the `ibc-go` codebase.
```

Comment:
```suggestion
This document details the acceptance process for requests from external contributors who require for the `ibc-go` core team to take over **maintainership** of a complex block of code (ie: an application module or a light client implementation). It is the process we also follow internally for features that we develop which will go into the `ibc-go` codebase.
```

Reply:
or different wording. I think there should be a stronger requirement than desiring for someone else to take over responsibility for a piece of code. We should encourage the community not to be reliant on us